src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 50px 0;
+  background-color: white;
+}
+
+.voteInfoContainer {
+  display: flex;
+  width: 85%;
+}
```

Comment:
The approve/dissaprove icon should be aligned with the avatar icons.

(yes, I know that the specs are mis-aligned as well, but that doesn't mean we have to implement them as broken)

![Screenshot from 2021-03-12 12-23-42](https://user-images.githubusercontent.com/1193222/110927166-f6a97600-832d-11eb-86eb-ade8c56bac03.png)

Reply:
Checked this after changing the approve/disapprove icons to the ones you provided and they seem to be aligned. I'm gonna mention anyway that in that screenshot the upper line it's not taking into account the `1px white border` that the avatars have

Reply:
Ok. I'll have a look again, taking the white border into account